common/build/build-common/tsconfig.test.json

```diff
@@ -0,0 +1,10 @@
+{
+	"extends": ["./tsconfig.esm.json"],
+	"compilerOptions": {
+		"composite": false,
+		"types": ["node", "mocha"],
```

Comment:
I'm assuming this was an existing pattern, but I don't think we should include `mocha` here. Many of our packages (mainly examples) use `jest`, rather than mocha. Seems like we should let the packages include their own required types based on the testing framework they're using IMO.

Reply:
Removed.